### ISSUE TITLE
Daily Telegraph and The Australian - cannot read readers' comments

### ISSUE THREAD

**Issue by AnthonyTheKoala** [johnbirds/bypass-paywalls-chrome-clean-magnolia1234#5]
Here are examples from the "Daily Telegraph". When you click "Read Comments" at the end of each article, the page refreshes but the readers' comments are not shown.

Try these and click "Read Comments":
https://amp.dailytelegraph.com.au/entertainment/sydney-confidential/2sm-radio-host-marcus-paul-dismissed-following-investigation/news-story/529fc2de4dfd45b12e4242b6e7f845fe

https://amp.dailytelegraph.com.au/business/unemployment-could-fall-to-3pc-rba/news-story/22ecd3143e9d1fdcf43df0c1a28b318a

https://amp.theaustralian.com.au/world/the-times/pope-tells-nuns-to-fight-back-against-sexist-priests/news-story/5b6c31967effa9a65e3008ea5a3edbc1

In summary: click on "Read Comments", the page refreshes but does not show readers' comments.
